common/web/input-processor/build.sh

```diff
@@ -131,31 +95,11 @@ fi
 
 if builder_start_action build :module; then
   npm run tsc -- -b src/tsconfig.json
-
   builder_finish_action success build :module
 fi
 
 # TEST ACTIONS
 
-if builder_start_action test :tools; then
-  builder_finish_action success test :tools
-fi
-
-if builder_start_action test :libraries; then
-  CHAINING_FLAGS=
-  if builder_has_option --ci; then
-    CHAINING_FLAGS=--ci
-  fi
-  # First, run tests on the keyboard processor.
-  pushd "$KEYMAN_ROOT/common/web/keyboard-processor"
-  ./build.sh test $CHAINING_FLAGS
-  popd
-
-  # lm-layer tests are handled outside of this chain, at least by our existing CI processes.
-
-  builder_finish_action success test :libraries
```

Comment:
What is launching these tests for CI builds after these changes?  I don't see where this call was moved anywhere.

Interestingly, you've removed this call here (and the `:libraries` target), but not from `common/predictive-text/unit_tests/test.sh`.

Reply:
So for now, common/predictive-text/unit_tests/test.sh is a top-level test script, rather than a submodule. Once we get a bit further on refactoring existing scripts (probably, once we have done _all_ the web-world build scripts), we can revisit how all the unit tests are triggered, because we may be able to come up with a good scheme that uses the dependency tree. But no point in trying to do that too soon...